src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -508,62 +508,62 @@ f1_429_trans_aff,40,2009,,,,,
-f1_accumdepr_prvsn,15,1994,Other Debit or Cr. Items (Describe):,,,,
-f1_accumdepr_prvsn,16,1994,Line 16,,,,
-f1_accumdepr_prvsn,17,1994,"Balance End of Year (Enter Totals of lines 1, 9, 14, 15, and 16)",,,,
-f1_accumdepr_prvsn,18,1994,Steam Production,,,,
-f1_accumdepr_prvsn,19,1994,Nuclear Production,,,,
-f1_accumdepr_prvsn,20,1994,Hydraulic Production-Conventional,,,,
-f1_accumdepr_prvsn,21,1994,Hydraulic Production-Pumped Storage,,,,
-f1_accumdepr_prvsn,22,1994,Other Production,,,,
-f1_accumdepr_prvsn,23,1994,Transmission,,,,
-f1_accumdepr_prvsn,24,1994,Distribution,,,,
-f1_accumdepr_prvsn,25,1994,General,,,,
-f1_accumdepr_prvsn,26,1994,TOTAL (Enter Total of lines 18 thru 25),,,,
-f1_accumdepr_prvsn,7,2002,"Other Accounts (Specify, details in footnote):",,,,
-f1_accumdepr_prvsn,15,2002,"Other Debit or Cr. Items (Describe, details in footnote):",,,,
-f1_accumdepr_prvsn,4,2003,(403.1) Depreciation Expense for Asset Retirement Costs,,,,
-f1_accumdepr_prvsn,5,2003,(413) Exp. of Elec. Plt. Leas. to Others,,,,
-f1_accumdepr_prvsn,6,2003,Transportation Expenses-Clearing,,,,
-f1_accumdepr_prvsn,7,2003,Other Clearing Accounts,,,,
-f1_accumdepr_prvsn,8,2003,"Other Accounts (Specify, details in footnote):",,,,
-f1_accumdepr_prvsn,9,2003,Line 8,,,,
-f1_accumdepr_prvsn,10,2003,TOTAL Deprec. Prov for Year (Enter Total of lines 3 thru 9),,,,
-f1_accumdepr_prvsn,11,2003,Net Charges for Plant Retired:,,,,
-f1_accumdepr_prvsn,12,2003,Book Cost of Plant Retired,,,,
-f1_accumdepr_prvsn,13,2003,Cost of Removal,,,,
-f1_accumdepr_prvsn,14,2003,Salvage (Credit),,,,
-f1_accumdepr_prvsn,15,2003,TOTAL Net Chrgs. for Plant Ret. (Enter Total of lines 12 thru 14),,,,
-f1_accumdepr_prvsn,16,2003,"Other Debit or Cr. Items (Describe, details in footnote):",,,,
-f1_accumdepr_prvsn,17,2003,Line 16,,,,
-f1_accumdepr_prvsn,18,2003,Book Cost or Asset Retirement Costs Retired,,,,
-f1_accumdepr_prvsn,19,2003,"Balance End of Year (Enter Totals of lines 1, 10, 15, 16, and 18)",,,,
-f1_accumdepr_prvsn,20,2003,Steam Production,,,,
-f1_accumdepr_prvsn,21,2003,Nuclear Production,,,,
-f1_accumdepr_prvsn,22,2003,Hydraulic Production-Conventional,,,,
-f1_accumdepr_prvsn,23,2003,Hydraulic Production-Pumped Storage,,,,
-f1_accumdepr_prvsn,24,2003,Other Production,,,,
-f1_accumdepr_prvsn,25,2003,Transmission,,,,
-f1_accumdepr_prvsn,26,2003,Distribution,,,,
-f1_accumdepr_prvsn,27,2003,General,,,,
-f1_accumdepr_prvsn,28,2003,TOTAL (Enter Total of lines 20 thru 27),,,,
-f1_accumdepr_prvsn,27,2006,Regional Transmission and Market Operation,,,,
-f1_accumdepr_prvsn,28,2006,General,,,,
-f1_accumdepr_prvsn,29,2006,TOTAL (Enter Total of lines 20 thru 28),,,,
+f1_accumdepr_prvsn,1,1994,Balance Beginning of Year,reported_value,starting_balance,,"Corresponds to accumulated_provision_for_depreciation_of_electric_utility_plant in XBRL instant table, but here we have to differentiate between the starting and 2 separate copies of the ending balances."
```

Comment:
There are some weird mappings in here, see the notes.  E.g. the starting balance and 2 separate (hopefully identical) ending balances, all of which correspond to the one value reported in the instant XBRL table, because of the reshaping that has to happen to create distinct `starting_balance` and `ending_balance` and a desire to be able to distinguish between the separately reported `ending_balance` for functional classifications vs. plant/account sections.

Reply:
i think i need to talk through this to understand what is going on here.

in general i think it is fine if these labels don't correspond to xbrl_factoid's per-say but do correspond to categories in the end table... but i'm a little confused about the layers of category columns in this table and how it relates to these mappings. these are all `ferc_account_label`'s... (which makes start/end balance a little weird). but these labels correspond to the stacked xbrl column stems? including start/end balance bc you invert the name in that process instant rename?

Reply:
Get rid of the  "account" in `ending_balance_account` column name in the account oriented table in the end (only meaningful in the context of the mapping, not the table)

Reply:
Removed this spurious "account" in the `instant_xbrl` rename.

Still not sure what these accounting categories should be called, since only a few of them are FERC accounts.